lib/client.js

```diff
@@ -4,9 +4,18 @@ const url = require('url');
 const urlJoin = require('url-join');
 const pkg = require('../package.json');
 
+const telemetryHeader = {
+  name: 'express-oidc',
+  version: pkg.version,
+  env: {
+    node: process.version
+  }
+};
+
 custom.setHttpOptionsDefaults({
   headers: {
-    'User-Agent': `${pkg.name}/${pkg.version} (${pkg.homepage})`
+    'User-Agent': `${pkg.name}/${pkg.version}`,
```

Comment:
Is this a good user agent? I removed the homepage, seems like unnecessary data.

Reply:
@joshcanhelp Do you care about Node version or possibly the OS?

If not, package name and version is probably enough.

Reply:
Node version is sent in the `env` object. We don't parse the user agent (at this point) for telemetry.